<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TickShiftWindow</class>
 <widget class="QDialog" name="TickShiftWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>146</height>
   </rect>
  </property>
  <widget class="QLabel" name="labelPrompt">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>201</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Shift item with :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="textInput">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>151</width>
     <height>38</height>
    </rect>
   </property>
   <property name="text">
    <string>2000</string>
   </property>
  </widget>
  <widget class="QFrame" name="frameRadioButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QRadioButton" name="radioSecond">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;second</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioMSecond">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>140</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;millisecond</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioMinute">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>mi&amp;nute</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frameActionButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>351</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="buttonOK">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>122</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonCancel">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>0</y>
      <width>122</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
